templates/typescript/cloudevents/src/index.ts

```diff
@@ -18,17 +18,17 @@ import { Context, Invokable } from 'faas-js-runtime';
  * See: https://github.com/knative-sandbox/kn-plugin-func/blob/main/docs/guides/nodejs.md#the-context-object
  * @param {CloudEvent} cloudevent the CloudEvent
  */
-export const handle: Invokable = function (
+const handle = function (
   context: Context,
-  cloudevent?: CloudEvent
+  cloudevent?: CloudEvent<string> | CloudEvent<Customer>
```

Comment:
I don't know TypeScript, I am just curious: would `cloudevent?: CloudEvent<string | Customer>` work?

Reply:
Actually it would. Nice catch.